cnspec-lint/README.md

```diff
@@ -0,0 +1,42 @@
+# Mondoo cnspec lint Action
+
+A [GitHub Action](https://github.com/features/actions) for testing [HashiCorp Terraform](https://terraform.io) [HCL code](https://www.terraform.io/language/syntax/configuration) for security misconfigurations.
+
+## Properties
+
+The cnspec lint Action has properties that are passed to the action using `with`.
+
+| Property      | Required | Default         | Description                                           |
+| ------------- | -------- | --------------- | ----------------------------------------------------- |
+| `path`        | true     | `.`             | Specifies the root path of the bundles .              |
+| `output-path` | true     | `results.sarif` | Specifies the output path for the sarif report file'. |
```

Comment:
If I look at the example below it seems like this should be `output` instead of `output-path`

Reply:
Perhaps it's also good to mention that this is not just a file to a folder but it should also include the filename

Reply:
Output is hardcoded to sarif, which is hard-coded. Right now the action only offers Sarif output to the `output-path`